src/memray/_memray/tracking_api.cpp

```diff
@@ -913,6 +890,35 @@ Tracker::unregisterPymallocHooks() const noexcept
 
 // Trace Function interface
 
+PyObject*
+create_profile_arg()
+{
+    // Borrowed reference
+    PyObject* memray_ext = PyDict_GetItemString(PyImport_GetModuleDict(), "memray._memray");
```

Comment:
`PyImport_GetModuleDict` cannot return `NULL` IIRC but it would be good to add an assert that's not NULL just in case

Reply:
It cannot, it raises a fatal error in that case instead: https://github.com/python/cpython/blob/6fd4c8ec7740523bb81191c013118d9d6959bc9d/Python/import.c#L253-L261

I believe that case could happen if something tries to spawn a new thread late in finalization while Memray tracking is still somehow enabled - but a fatal error from the interpreter is a nice way to handle it, anyway.